libs/shared/src/consts/providers.ts

```diff
@@ -0,0 +1,17 @@
+export const providers = [
+  'mailjs',
```

Comment:
Hey i was not sure if the providers data will be available soon on the DB or not. If it wont it will be really more informative to add additional information on here as you pointed above.